Forge Networking Remastered Unity/Assets/Bearded Man Studios Inc/Scripts/Multiplayer Menu/MultiplayerMenu.cs

```diff
@@ -74,11 +74,16 @@ public void Connect()
 			ConnectToMatchmaking();
 			return;
 		}
-		int port = ushort.Parse(portNumber.text);
-		if (port < 0 || port > ushort.MaxValue)
+		int port = -1;
```

Comment:
Thanks for this Pull request :). Can you use the `ushort.TryParse` method for this and in the failed case do what you are doing for the catch?